code/modules/client/lightmode.css

```diff
@@ -85,7 +85,7 @@ h1.alert, h2.alert		{color: #000000;}
 .info					{color: #0000cc;}
 .notice					{color: #000099;}
 .subtle					{color: #000099; font-size: 75%; font-style: italic;}
-.alium					{color: #00ff00;}
+.alium					{color: #439C16;}
```

Comment:
That's a css file. So that's probably why it's not tested. You'd have to check the loaded css string that's injected into the UI to validate that.

Reply:
I could probably change it to lowercase if it's really a problem?
Also, another thing I noticed, is that, this reaally shouldn't be assumed to be CSS, because byond's style attribute doesn't support a lot of what css does? Like if you use hsla() to specify a color, it just won't get parsed.

Reply:
hsla was first supported in IE9, byond's internal css handling has partial parity with like. IE6 or something. don't expect things newer than IE7 to work reliably

Reply:
also technically the reason we do lowercase in dm files is because if you do `color = "#FF0000"; color == "#FF0000"` it'll be false because it's internally converted to `"#ff0000"`. not technically necessary for css but it's still a preferred stylistic thing

Reply:
Alright, that explains it.

Reply:
Sounds good, I'll make the change.

Reply:
Alright, I made the change, and added some comments to the css files to make it clear it doesn't support the same css subset as everything else. And why the color hex values are lowercase.